services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    else:
+        queryset = queryset.all()
```

Comment:
The `else` block with `queryset = queryset.all()` is redundant because `queryset` is already initialized with `Movie.objects.all()`. You can remove this line to simplify the code.